cmd/kops/delete.go

```diff
@@ -30,7 +31,7 @@ func NewCmdDelete(f *util.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:        "delete",
-		Short:      "delete clusters",
+		Short:      "Delete clusters and other resources.",
```

Comment:
What other resources? Resources associated to the cluster? Not sure how to say that short and sweet.